lib/libzutil/os/windows/zutil_import_os.c

```diff
@@ -321,7 +321,7 @@ zpool_open_func(void *arg)
 	 */
 	error = nvlist_lookup_uint64(config, ZPOOL_CONFIG_GUID, &vdev_guid);
 	if (error || (rn->rn_vdev_guid && rn->rn_vdev_guid != vdev_guid)) {
-		(void) close(fd);
+		(void) close(HTOI(fd));
 		nvlist_free(config);
```

Comment:
OK, so I wrote this, but, all other calls just use CloseHandle(), so having just one close() in here is a bit weird. and we could also discuss using "fd" as "HANDLE fd". Perhaps fd should only be "file descriptors" and we use something else ("h"? "handle"?) in the windows-only code

Reply:
Not only that but you use CloseHandle just below this

Reply:
Done with fd -> h